extensions/resource-deployment/package.nls.json

```diff
@@ -20,6 +20,9 @@
 	"bdc-2019-aks-notebook": "./notebooks/bdc/2019/deploy-bdc-aks.ipynb",
 	"bdc-2019-existing-aks-notebook": "./notebooks/bdc/2019/deploy-bdc-existing-aks.ipynb",
 	"bdc-2019-existing-kubeadm-notebook": "./notebooks/bdc/2019/deploy-bdc-existing-kubeadm.ipynb",
+	"azdata-bdc-2019-aks-notebook": "./notebooks/bdc/2019/azdata/deploy-bdc-aks.ipynb",
```

Comment:
Just noticed this - but why are these in the loc file? Do you actually expect these to ever be localized?

Reply:
good question. we don't have a way to localize the string in notebooks, I added the path here so that we could localize the notebook itself. but I don't think our localization process works like this. I'll start an Email thread for this.